A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -762,19 +762,23 @@ class CDUPerformancePage {
                 CDUPerformancePage.ShowAPPRPage(mcdu);
             }
         };
-
         let transAltCell = "[\xa0][color]cyan";
-        if (isFinite(mcdu.perfApprTransAlt)) {
-            transAltCell = mcdu.perfApprTransAlt.toFixed(0);
-            if (!mcdu.perfApprTransAltPilotEntered) {
-                transAltCell += "[s-text]";
+        if (hasDestination) {
```

Comment:
Why are you even changing anything on approach page? You haven't implemented transition level in this PR? I should be able to figure it out from the commit history, but it's incoherent on this PR.

Reply:
What? Approach page is TRNASITION ALTITUDE

Reply:
There seems to be a bit of debate amongst pilots in the threads I can find on pprune etc. I suspect it may depend whose navdata you have. There is also a bit of discussion of the label being changed to trans lvl in the latest revisions but hard to know if that's Thales, Honeywell, or both and I have no good references.

Anyway, loading trans alt wouldn't seem to be of much use to me e.g. in NZ with TA = 13000 and TL = FL150, it's a bit late to blink the QNH reminder at 13000 feet on descent, I want that at FL150 (which is 15000 feet with baro = STD). Most airline SOPs are to switch to STD when cleared to an FL during climb and back to QNH when cleared to an altitude on descent so it's probably quite rare for pilots to actually see the reminder.

-e- We do have a reference from one of our own pilots:
![79B48E11-878F-400B-931B-D2686E257CD1](https://user-images.githubusercontent.com/9995998/111968558-24789100-8b5e-11eb-8c53-8ceee1ddb518.jpeg)

Reply:
yes :) I provided reference video by him.
so, i'll add Trans LVL logic (soon) but at current, we use trans alt :(.

please wait until i make new db for trans lvl & calculate logic(or maybe with EFB option?

btw, Idk there was a trans lvl in appr phase. maybe it is different by honeywell & thales and even dependent version i think?